grpc-proto/proto/stargate.proto

```diff
@@ -29,5 +30,8 @@ service Stargate {
 
   // Executes a batch of CQL queries.
   rpc ExecuteBatch(Batch) returns (Response) {}
+
+  // Requests notification of schema changes.
+  rpc getSchemaChanges(google.protobuf.Empty) returns (stream SchemaChange) {}
```

Comment:
We might want to use a custom message here. Even if it's still empty now, that leaves us a way to add parameters in the future if needed.

Reply:
Done.